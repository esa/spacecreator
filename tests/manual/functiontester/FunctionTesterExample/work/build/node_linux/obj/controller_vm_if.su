controller_vm_if.c:14:6:init_controller	32	static
controller_vm_if.c:32:6:controller_Trigger	32	static
controller_vm_if.c:48:1:_GLOBAL__sub_D_00099_0_controller_initialized	16	static
controller_vm_if.c:48:1:_GLOBAL__sub_I_00099_1_controller_initialized	16	static
