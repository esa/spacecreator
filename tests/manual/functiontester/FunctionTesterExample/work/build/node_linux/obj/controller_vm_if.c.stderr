[01m[K/home/taste/FunctionTesterExample/work/controller/C/wrappers/controller_vm_if.c:[m[K In function â€˜[01m[Kinit_controller[m[Kâ€™:
[01m[K/home/taste/FunctionTesterExample/work/controller/C/wrappers/controller_vm_if.c:18:17:[m[K [01;35m[Kwarning: [m[Kunused variable â€˜[01m[Kunused[m[Kâ€™ [[01;35m[K-Wunused-variable[m[K]
       long long [01;35m[Kunused[m[K = getTimeInMilliseconds();
                 [01;35m[K^~~~~~[m[K
